pkg/apis/core/v1alpha1/types_project.go

```diff
@@ -61,6 +61,7 @@ type ProjectSpec struct {
 	// Owner is a subject representing a user name, an email address, or any other identifier of a user owning
 	// the project.
 	// +optional
+	// TODO: Remove this field in favor of the `owner` role in `v1`.
```

Comment:
Can you add a deprecation note as well?

Reply:
The field is not deprecated yet, hence, I didn't add a deprecation note. Instead, I described that it will be removed in the next version so that people that are using the `owner` field now are aware.